webui/react/src/pages/ExperimentDetails/ExperimentCheckpoints.settings.ts

```diff
@@ -118,3 +118,81 @@ export const configForExperiment = (id: number): SettingsConfig<Settings> => ({
   },
   storagePath: `${id}-checkpoints`,
 });
+
+export const configForTrial = (id: number): SettingsConfig<Settings> => ({
+  settings: {
+    columns: {
+      defaultValue: DEFAULT_COLUMNS,
+      skipUrlEncoding: true,
+      storageKey: 'columns',
+      type: array(
+        union([
+          literal('action'),
+          literal('uuid'),
+          literal('state'),
+          literal('searcherMetric'),
+          literal('totalBatches'),
+          literal('checkpoint'),
+        ]),
```

Comment:
nit: this will do a linear search for values in the type -- use the `keyof` codec from `io-ts` instead:
```suggestion
        keyof({
          action: null,
          checkpoint: null,
          searcherMetric: null,
          state: null,
          totalBatches: null,
          uuid: null,
        }),
```